pkg/registry/file/storage.go

```diff
@@ -24,21 +26,19 @@ import (
 )
 
 const (
-	defaultChanSize    = 100
 	jsonExt            = ".json"
 	metadataExt        = ".metadata"
 	DefaultStorageRoot = "/data"
 )
 
 type objState struct {
-	obj   runtime.Object
-	meta  *storage.ResponseMeta
-	rev   int64
-	data  []byte
-	stale bool
+	obj  runtime.Object
+	meta *storage.ResponseMeta
+	rev  int64
+	data []byte
```

Comment:
question: is the `stale bool` variable no longer needed? Why? Was it used before?

Reply:
I think it was never needed, golangci-lint was complaining.